foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_10_0/JavaPackageMigration.java

```diff
@@ -0,0 +1,89 @@
+package com.boozallen.aissemble.upgrade.migration.v1_10_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+
+/**
+ * Updates the affected java classes from their old package name to their new package name 
+ * to ensure compatibility with the updated Java 17 dependencies 
+ */
+public class JavaPackageMigration extends AbstractAissembleMigration {
+    private static final Logger logger = LoggerFactory.getLogger(JavaPackageMigration.class);
+    private static final String[][] JAVA_PACKAGES = {
```

Comment:
A: I feel like this becomes easier to track in the actual logic as a `Map.of` instead of a list of lists.

Reply:
Updated to use maps. Fun fact, `Map.of` only takes 10 entries max, while `Map.ofEntries` takes unlimited